locales-src/newtab.ftl

```diff
@@ -0,0 +1,159 @@
+newtab-topsites-cancel-button = Cancel
+newtab-topsites-delete-history-button = Delete from History
+newtab-topsites-save-button = Save
+newtab-topsites-preview-button = Preview
+newtab-topsites-add-button = Add
+
+## Top Sites - Delete history confirmation dialog. newtab-confirm-delete-history-p1 appears in the same dialog
+## as newtab-confirm-delete-history-p2.
+newtab-confirm-delete-history-p1 = Are you sure you want to delete every instance of this page from your history?
+# Note: "This action" refers to deleting a page from history.
+newtab-confirm-delete-history-p2 = This action cannot be undone.
+
+newtab-topsites-add-search-engine = Add Search Engine
+
+## Context Menu - Action Tooltips.
+# General tool tip for context menus.
+newtab-menu-section-tooltip =
+    .title = Open menu
+    .aria-label = Open menu
+
+# This tooltip is for the context menu of Pocket cards or Topsites
+# Variables:
+#  $title (String): Title is the label or hostname of the site. This is for screen readers when the context menu button is focused/active.
+newtab-menu-content-tooltip =
+    .title = Open menu
+    .aria-label = Open context menu for { $title }
+# Tooltip on an empty topsite box to open the New Top Site dialog.
+newtab-menu-topsites-placeholder-tooltip =
+    .title = Edit this site
+    .aria-label = Edit this site
+
+## Context Menu: These strings are displayed in a context menu and are meant as a call to action for a given page.
+newtab-menu-edit-topsites = Edit
+newtab-menu-open-new-window = Open in a New Window
+newtab-menu-open-new-private-window = Open in a New Private Window
+newtab-menu-dismiss = Dismiss
+newtab-menu-pin = Pin
+newtab-menu-unpin = Unpin
+newtab-menu-delete-history = Delete from History
+newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
+newtab-menu-delete-pocket = Delete from { -pocket-brand-name }
+newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+
+# Note: Bookmark is a noun in this case, "Remove bookmark".
+newtab-menu-remove-bookmark = Remove Bookmark
+# Note: Bookmark is a verb here.
+newtab-menu-bookmark = Bookmark
+
+## Context Menu - Downloaded Menu: In (newtab-menu-copy-download-link, newtab-menu-go-to-download-page), 
+## "Download" in these cases is not a verb, it is a noun. As in, "Copy the link that belongs to this downloaded item".
+newtab-menu-copy-download-link = Copy Download Link
+newtab-menu-go-to-download-page = Go to Download Page
+newtab-menu-remove-download = Remove from History
+
+## Context Menu - Download Menu: These are platform specific strings found in the context menu of an item that has
+## been downloaded. The intention behind "this action" is that it will show where the downloaded file exists on the file
+## system for each operating system.
+newtab-menu-show-file =
+    { PLATFORM() ->
+        [macos] Show in Finder
+        [windows] Open Containing Folder
+        [linux] Open Containing Folder
+       *[other] Show File
```

Comment:
The spec https://mozilla.invisionapp.com/share/J6GEU4EHTYC#/screens and comments in the original bug https://bugzilla.mozilla.org/show_bug.cgi?id=1433230 don't seem to see any mentions now (spec might have been updated since then?) of "Show File". So yeah combining them for special mac and everything else seems good.